libs/pilight/core/common.c

```diff
@@ -111,7 +112,7 @@ void array_free(char ***array, int len) {
 	}
 }
 
-unsigned int explode(char *str, const char *delimiter, char ***output) {
+unsigned int explode(const char *str, const char *delimiter, char ***output) {
```

Comment:
BTW, your code is a good example of whats happen if code tries to break the 'const' promise:
```c
const char *a = "foo";  // its promised that further code won't modify chars that a points to.
char *p = a;    // compiler warns that 'const' attribute is dropped (this breaks the promise because p could be used to modify chars that a points to).
```